pkg/api/fuzzer/fuzzer.go

```diff
@@ -396,14 +396,16 @@ var Funcs = func(codecs runtimeserializer.CodecFactory) []interface{} {
 			}
 		},
 		func(sio *api.ScaleIOVolumeSource, c fuzz.Continue) {
-			sio.ProtectionDomain = c.RandString()
-			if sio.ProtectionDomain == "" {
-				sio.ProtectionDomain = "default"
+			sio.StorageMode = c.RandString()
+			if sio.StorageMode == "" {
+				sio.StorageMode = "ThinProvisioned"
 			}
-			sio.StoragePool = c.RandString()
-			if sio.StoragePool == "" {
-				sio.StoragePool = "default"
+			sio.FSType = c.RandString()
+			if sio.FSType == "" {
+				sio.FSType = "xfs"
 			}
```

Comment:
why did ProtectionDomain and StoragePool get dropped? should be present in both fuzzing funcs, right?

Reply:
@liggitt the assumption that these two params would default to value `default` was a bug.  Because if those default were actually applied the volume would fail.  This was partially fixed in the last release.  So I removed them here in accordance with what was done then (https://github.com/kubernetes/kubernetes/pull/49973).

Reply:
hmm, ok. changing defaults is typically considered a breaking API change, but if the defaults caused the object to never work, that doesn't seem good either.

would like agreement from @kubernetes/sig-storage-api-reviews on the removal of the defaulting behavior on those fields, and if agreed on, remove the ` (defaults to "default")` godoc from those fields and add a description of the default change to the release note.

Reply:
@liggitt thank you. Basically this is taking care of a bug in the API validation.  If those values are applied by default as `default`, chances are they would break storage operations since, most likely, these values will not match values on the backing storage system.

Reply:
Ok. Validation doesn't require those values to be set (and can't, without breaking existing API create calls), so with defaulting removed, if someone leaves them empty, they will just be unset now. Is that acceptable?

Reply:
@liggitt Yes.  That is acceptable.